code/datums/traits/negative.dm

```diff
@@ -154,3 +154,12 @@
 	else if(prob(0.5) && dumb_thing)
 		to_chat(H, "<span class='danger'>You think of a dumb thing you said a long time ago and scream internally.</span>")
 		dumb_thing = FALSE //only once per life
+
+/datum/trait/dnr
```

Comment:
Might want to move this to the top of the file, as entries are listed by the order of the file and they're currently ordered alphabetically.